db/init.sql

```diff
@@ -4,13 +4,315 @@ CREATE DATABASE mm_db;
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
 CREATE TYPE user_role AS ENUM ('admin', 'teacher', 'student');
+CREATE TYPE attempt_state AS ENUM ('in_progress', 'completed');
```

Comment:
```suggestion
CREATE TYPE attempt_state AS ENUM ('submitted', 'graded');
```

In my opinion, these namings seem more appropriate for determining the attempt state, since the attempt was already "completed" at the time it was sent

Reply:
Agree with you.